### PR TITLE
[16.0][MIG] account_asset_batch_compute: Migration to 16.0

### PR DESCRIPTION
**Pull request by JavierIniesta**
Migration to 16.0

#1472

### CODE REVIEW

File: account_asset_batch_compute/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Adrien Peiffer
+* Jordi Ballester Alomar <jordi.ballester@forgeflow.com>
+* Alba Riera <alba.riera@creublanca.es>
```
Comment (ljsalvatierra-factorlibre) [inline account_asset_batch_compute/readme/CONTRIBUTORS.rst:3]: I think you should be here.

**Review by ljsalvatierra-factorlibre — CHANGES_REQUESTED** [review]

**Comment by JavierIniesta** [comment]
We have done functional tests and it works correctly.

**Review by RodrigoBM — APPROVED** [review]

**Review by ljsalvatierra-factorlibre — APPROVED** [review]

File: account_asset_batch_compute/wizards/account_asset_compute.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class AccountAssetCompute(models.TransientModel):
+
+    _inherit = "account.asset.compute"
+
+    batch_processing = fields.Boolean()
+
+    def asset_compute(self):
+        self.ensure_one()
+        if not self.batch_processing:
+            return super(AccountAssetCompute, self).asset_compute()
```
Comment (RodrigoBM) [inline account_asset_batch_compute/wizards/account_asset_compute.py]: Apply PEP 3135 (28-Apr-2007) to "super()."

**Review by RodrigoBM — CHANGES_REQUESTED** [review]

File: account_asset_batch_compute/wizards/account_asset_compute.py (hunk 2 above)
Comment (JavierIniesta) [reply account_asset_batch_compute/wizards/account_asset_compute.py]: Done.

**Review by RodrigoBM — APPROVED** [review]

**Comment by RodrigoBM** [comment]
is it possible to merge this PR? @OCA/accounting-maintainers

**Pull request merged**